packages/framework-core/test/booster.test.ts

```diff
@@ -253,4 +254,47 @@ describe('the `Booster` class', () => {
       })
     })
   })
+
+  describe('The `loadTokenVerifierFromEnv` function', () => {
+    context('when the JWT_ENV_VARS are set', () => {
+      beforeEach(() => {
+        process.env.BOOSTER_JWT_ISSUER = 'BOOSTER_JWT_ISSUER_VALUE'
+        process.env.BOOSTER_JWKS_URI = 'BOOSTER_JWKS_URI_VALUE'
+        process.env.BOOSTER_ROLES_CLAIM = 'BOOSTER_ROLES_CLAIM_VALUE'
+      })
+
+      afterEach(() => {
+        delete process.env.BOOSTER_JWT_ISSUER
+        delete process.env.BOOSTER_JWKS_URI
+        delete process.env.BOOSTER_ROLES_CLAIM
+
+        Booster.config.tokenVerifiers = []
+      })
```

Comment:
At some point we should start injecting `process.env` into Booster rather than relying on the variables themselves directly in the codebase, in order to make it more testable

Reply:
👍🏼